GATT.py

```diff
@@ -256,11 +260,11 @@ def GetAll(self, interface):
         if interface != constants.GATT_DESCRIPTOR_INTERFACE:
             raise exceptions.InvalidArgsException()
 
-        return self.get_properties()[constants.GATT_DESCRIPTOR_INTERFACE]
+        return self.get_properties()[interface]
 
     @dbus.service.method(constants.GATT_DESCRIPTOR_INTERFACE,
                         in_signature='a{sv}',
-                        out_signature='ay')
+                        out_signature='')
```

Comment:
```suggestion
                        out_signature='ay')
```

IDK what this does. Gonna put it back to test.